frida_tools/tracer.py

```diff
@@ -105,16 +110,19 @@ def on_trace_error(self, error):
 
         def on_trace_events(self, events):
             no_attributes = Style.RESET_ALL
-            for timestamp, thread_id, depth, target_address, message in events:
-                if self._quiet:
-                    self._print(message)
-                else:
-                    indent = depth * "   | "
-                    attributes = self._get_attributes(thread_id)
-                    if thread_id != self._last_event_tid:
-                        self._print("%s           /* TID 0x%x */%s" % (attributes, thread_id, Style.RESET_ALL))
-                        self._last_event_tid = thread_id
-                    self._print("%6d ms  %s%s%s%s" % (timestamp, attributes, indent, message, no_attributes))
+            with OutputFile(self._output) as output:
+                for timestamp, thread_id, depth, target_address, message in events:
+                    if output is not None:
+                        output.append(message + "\n")
+                    elif self._quiet:
```

Comment:
I guess this has to be is not None comparison to be consistent?

Reply:
Seems like store_true makes the python option parsing use None or True instead of False|True. I have fixed this before caching that option inside the object.

Reply:
How about giving it a default value of `False` instead – won't that make it `False|True`?

Reply:
my last commit fixed this already